package.json

```diff
@@ -41,20 +41,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/NEAR-DevHub/neardevhub-trustees-dashboard-bos"
+    "url": "git+https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard.git"
   },
   "keywords": [
     "near",
     "nearprotocol",
     "bos",
-    "blockchains",
+    "blockchainos",
     "typescript",
-    "trustees"
+    "starter"
   ],
-  "author": "devhub.near",
+  "author": "frol.near",
```

Comment:
Let's revert it to devhub.near (I'll need to fix it in the template, so I don't suddenly get too many projects created by me)